src/AppBridgeBlock.ts

```diff
@@ -458,7 +453,7 @@ export class AppBridgeBlock {
         };
     }
 
-    private mapDocumentBlockAssetApiToFrontifyAsset(documentBlockAsset: DocumentBlockAssetApi): FrontifyAsset {
+    private mapDocumentBlockAssetApiToFrontifyAsset(documentBlockAsset: DocumentBlockAssetApi): Asset {
```

Comment:
We should update the method name as well:
```suggestion
    private mapDocumentBlockAssetApiToAsset(documentBlockAsset: DocumentBlockAssetApi): Asset {
```